background_scripts/completion_search.coffee

```diff
@@ -42,7 +42,7 @@ CompletionSearch =
 
   # The amount of time to wait for new requests before launching the current request (for example, if the user
   # is still typing).
-  delay: 100
+  delay: 0
```

Comment:
This may not be a great idea.

I don't have a precise figure, but ...

If you turn on debugging in this file then you can watch what happens to completion requests.  A pretty significant portion (30%-50%, perhaps) of them are cancelled before being launched because the user is still typing.  As good netizens, there's no need to send that unneeded load to the servers.